### PR TITLE
add LICENSE file

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

**Comment by xabbuh** [comment]
Actually, I first just wanted to update the year in the license, but then I noticed that there was no license file at all. Choosing the MIT license here was done on purpose given that this is already stated in the `composer.json` file and that it is consistent with the rest of the Symfony codebase.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2014-2015 Fabien Potencier
```
Comment (xabbuh) [inline LICENSE]: @javiereguiluz I'm not completely sure here since afaik you initially wrote the first version of the installer.

Comment (javiereguiluz) [inline LICENSE]: I think it's fine to keep Fabien :)

**Comment by javiereguiluz** [comment]
Thanks Christian.

**Pull request merged**